components/should_verify_email/should_verify_email.tsx

```diff
@@ -1,131 +1,95 @@
-                        <div>
-                            <p data-testid='emailVerifyNotVerifiedBody'>
-                                <FormattedMessage
-                                    id='email_verify.notVerifiedBody'
-                                    defaultMessage='Please verify your email address. Check your inbox for an email.'
+    return (
+        <div className='should-verify-body'>
+            <div className='should-verify-body-content'>
+                <ColumnLayout
+                    title={formatMessage({id: 'email_verify.almost', defaultMessage: 'You’re almost done!'})}
+                    message={formatMessage({id: 'email_verify.notVerifiedBody', defaultMessage: 'Please verify your email address. Check your inbox for an email.'})}
+                    SVGElement={<ManWithMailboxSVG width={284}/>}
+                    extraContent={(
+                        <div className='should-verify-body-content-extra'>
+                            <div className='should-verify-body-content-buttons'>
+                                <SaveButton
+                                    extraClasses='should-verify-body-content-button-resend large'
+                                    saving={isWaiting}
+                                    disabled={!email}
+                                    onClick={handleResendButtonOnClick}
+                                    defaultMessage={formatMessage({id: 'email_verify.resend', defaultMessage: 'Resend Email'})}
+                                    savingMessage={formatMessage({id: 'email_verify.sending', defaultMessage: 'Sending email…'})}
                                 />
-                            </p>
-                            <button
-                                data-testid='emailVerifyResend'
-                                onClick={this.handleResend}
-                                className='btn btn-primary'
-                            >
-                                <FormattedMessage
-                                    id='email_verify.resend'
-                                    defaultMessage='Resend Email'
-                                />
-                            </button>
-                            {resendConfirm}
+                                <button
+                                    className='should-verify-body-content-button-return'
+                                    onClick={handleReturnButtonOnClick}
+                                >
+                                    {formatMessage({id: 'email_verify.return', defaultMessage: 'Return to log in'})}
+                                </button>
+                            </div>
+                            <div className={classNames('should-verify-body-content-message', resendStatus)}>
+                                <i className={classNames('should-verify-body-content-message-icon', 'icon', 'icon-12', resendStatus === 'success' ? 'icon-check' : 'icon-alert-outline')}/>
```

Comment:
nit: we should try to place these kind of strings like "success" in constants.

Reply:
Updated